packages/block-scripts/package.json

```diff
@@ -41,7 +41,6 @@
     "fs-extra": "^10.1.0",
     "html-webpack-plugin": "^5.5.0",
     "lodash": "^4.17.21",
-    "regenerator-runtime": "^0.13.9",
```

Comment:
Not sure we need this package. We don’t import/require it and all our targets support regenerators natively.